This fixes an audio synchronization bug when exporting videos from
kdenlive.  See http://www.kdenlive.org/mantis/view.php?id=28.

Ollie Wild <aloysius.wild@gmail.com>

--- mlt-0.2.4/src/modules/avformat/consumer_avformat.c.orig	2008-02-24 01:20:25.000000000 -0800
+++ mlt-0.2.4/src/modules/avformat/consumer_avformat.c	2008-02-24 01:20:38.000000000 -0800
@@ -623,7 +623,7 @@
 	struct timeval ante;
 
 	// Get the frame rate
-	int fps = mlt_properties_get_double( properties, "fps" );
+	double fps = mlt_properties_get_double( properties, "fps" );
 
 	// Get width and height
 	int width = mlt_properties_get_int( properties, "width" );
